issue_links.go

```diff
@@ -68,6 +69,7 @@ func (s *IssueLinksService) ListIssueRelations(pid interface{}, issueIID int, op
 //
 // GitLab API docs: https://docs.gitlab.com/ee/api/issue_links.html
 type CreateIssueLinkOptions struct {
+	LinkType        *string `json:"link_type"`
```

Comment:
Please place this field below the target fields (so following the order in the docs). Thanks!